docs-md/developer-guide/ksqldb-rest-api/ksql-endpoint.md

```diff
@@ -82,6 +84,10 @@ Response JSON Object:
 - **tables[i].topic** (string): The topic backing the table.
 - **tables[i].format** (string): The serialization format of the data in the
   table. One of JSON, AVRO, or DELIMITED.
+- **tables[i].type** (string): The source type. Will always be `TABLE`.
+
+
+- **tables[i].isWindowed** (boolean): `true` if the table provides windowed results; otherwise, `false`.

```

Comment:
```suggestion
- **tables[i].type** (string): The source type. Will always be `TABLE`.
- **tables[i].isWindowed** (boolean): `true` if the table provides windowed results; otherwise, `false`.
```